### ISSUE TITLE
Turning off Fan will reset Fan Schedule

### ISSUE THREAD

**Issue by randy3k** [chrisjshull/homebridge-nest#314]
**Describe the bug**

1. Set a Fan Schedule via the Nest App.
2. Turn on the Fan via Homekit.
3. Turn off the Fan via Homekit.
4. Observe that the Fan Schedule is now reset.

I am no the latest Homebridge and `homebridge-nest`. My thermostat is Nest Thermostat E.

**Comment by adriancable** [comment]
Hi @randy3k - sorry about this. It does sound like a bug. As you probably know we support two completely different thermostat APIs (REST and protobuf) and the protobuf API is newer and so may have some corner case bugs. I am wondering if your thermostat is on the protobuf API.

Could you edit lib/nest-device-accessory.js in your plug-in, and change line 45 from:

`this.log('initing ' + this.deviceType + ' "' + this.name + '":', 'deviceId:', this.deviceId, 'structureId:', this.structureId);`

To:

`this.log('initing ' + this.deviceType + ' "' + this.name + '":', 'deviceId:', this.deviceId, 'structureId:', this.structureId, device.using_protobuf);`

Then please restart Homebridge and post your log of it starting up (showing all the 'initing' lines for your Nest devices). We'll go from there.

**Comment by randy3k** [comment]
You are right. `device.using_protobuf` returns `true`.
```
[10/8/2020, 15:59:35] [Nest] initing thermostat "Hallway Thermostat": deviceId: 6416660000FB2951 structureId: e8c96ca1-c43b-11e9-a328-0af2a1358654 true
```

**Comment by adriancable** [comment]
@randy3k - can you email me at adrian.cable@gmail.com so we can resolve and test this together before I push an update?

**Comment by adriancable** [comment]
Fixed in 4.4.12.
